### PR TITLE
graphQL - fix1 login, register +jwt

### PR DESCRIPTION
**Pull request by Lazeratrax**
graphQL - fix1
+  login, register
+ jwt
+ fix buildSchema in "schema.js"

implemented boilerplate https://auth0.com/blog/build-and-secure-a-graphql-server-with-node-js/

### CODE REVIEW

File: src/database.js [hunk 1]
```diff
@@ -4,46 +4,76 @@ const {MongoClient} = require("mongodb")
...
+    // await database.collection("Output").insertMany([
+    //     {
+    //         param: 'firstName',
+    //         msg: 'Invalid value',
+    //     },
+    //     {
+    //         param: 'password',
+    //         msg: 'Invalid value',
+    //     }
+    // ])
```
Comment (ZubelRuslan) [inline src/database.js:75]: Better to create other function for this (like `insertInitialMockData`)

File: src/resolver.js [hunk 2]
```diff
@@ -24,99 +23,132 @@ const resolvers = {
...
     },
-    createUser: async ({id, firstName, password}, context) => {
+    createUser: async ({user: {id, name, password}}, context) => {
         const {db} = await context();
-        //хэширование пароля
         const hashPassword = await bcrypt.hash(password, 12);
-        const newUser = {id, firstName, password: hashPassword};
+        const newUser = {id, name, password: hashPassword};
+        console.log('New user: ', newUser)
         db.collection("users").insertOne(newUser);
```
Comment (ZubelRuslan) [inline src/resolver.js:31]: Why you didn't use mongoose models?

File: src/database.js (hunk 1 above)
Comment (Lazeratrax) [reply src/database.js:75]: ok, I must use any service, like https://www.mockaroo.com/ ? And insert generated data into 
"await database.collection("users").insertMany(["   ])"" ?

File: src/resolver.js (hunk 2 above)
Comment (Lazeratrax) [reply src/resolver.js:31]: Sadly, I haven't seen any examples project using both technologies: "graphql" & "mongoose models".. I thought it a little wrong;)

File: src/database.js (hunk 1 above)
Comment (ZubelRuslan) [reply src/database.js:75]: It isn't required. Only move this logic to separate function. Code style issue

**Pull request merged**